geniuspaste/src/geniuspaste.c

```diff
@@ -94,6 +94,54 @@ PLUGIN_SET_TRANSLATABLE_INFO(LOCALEDIR, GETTEXT_PACKAGE, PLUGIN_NAME,
                              PLUGIN_VERSION, "Enrico Trotta <enrico.trt@gmail.com>")
 
 
+/* Windows compatibility for resolving system plugin data path.
+ * This only works if the plugin's data is installed in the same tree as the
+ * plugin DLL, which might not be true (as the DLL has to be in Geany's libdir
+ * but the plugin's data might be anywhere in theory) */
```

Comment:
@codebrainz well, I don't want to pollute the user directory with stuff installed by the plugin that would better be off somewhere else.  If the user wants to edit those, he copies them in his own directory, but he shouldn't need to, it shuold Just Work.  But it's totally optional, I don't need them to function, without them there's just no default pastebins.